docs/applications/pyserver/index.rst

```diff
@@ -0,0 +1,16 @@
+Proxy-Server (python)
+=======================================
+
+- a Python server that runs on the Receivers machine
+
+
+
+.. toctree::
+   :maxdepth: 2 
+
+   installation
+   usage
+   cmdline
```

Comment:
This is empty when I build the docs - am I missing something? 
Btw, this is quite important for both pyserver and pyclient. Same for running the examples, which should also be documented.

Reply:
This will auto-document the click options, have to look into it.
What's the status on that, wasn't that not supported at the moment?

Reply:
> What's the status on that, wasn't that not supported at the moment?
What are you referring to?

Also, instructions for building the docs would also be nice. Maybe I missed something. I just did a `sphinx-build -b html docs build`

Reply:
In the other PR #417 that's still open (and somewhat redundant, sorry), you commented:

   Docs show:
   microraiden [OPTIONS] COMMAND [ARGS]... and
   microraiden start [OPTIONS]

   Not valid commands at this point.

You probably mean that microraiden doesn't have a `scripts` entrypoint registered during setup?

The above `cmdline` docs just use a click auto-documenting plugin, so I don't know how much customization is available.
For now we can just document and explai the args passed to the example scripts manually (`--private-key` etc). This has some overlap with the click auto-documentation though.

Reply:
The files `Installation.rst` and `usage.rst` are fine in my build. 
But in the file `../applications/pysercer/cmdline.rst` the section
```
.. click:: microraiden.click_helpers:main
   :prog: microraiden
   :show-nested:
```
doesn't compile correctly.

![zum beispiel](https://user-images.githubusercontent.com/3756671/36726210-4f6e7acc-1bb9-11e8-9fb4-7bf071a13eb3.png)

@ezdac could you help me out with this issue?